packages/plugin-sass/plugin.js

```diff
@@ -0,0 +1,23 @@
+const path = require("path");
+const sass = require("node-sass");
```

Comment:
I’m actually pretty 👎 on `node-sass`. Particularly with it having some pretty fundamental issues like these:

- https://github.com/sass/node-sass/issues/2886
- https://github.com/sass/node-sass/issues/2251

For those not familiar, this is **VERY** different than Dart Sass. Almost a different language. 🤔 I’m wondering if there’s a way to use Dart Sass’ CLI somehow in this plugin, which is the “official” distro of Sass now. That would certainly make this plugin not only more usable, but possibly faster, too.